fatehrai/MMED-1054-Biographical-App-Assignment
# Updated Biographical App

## What updates have been made?
* All the information/content about Phil has been added.
* Some of the text in Fateh's section has been rearranged/reworded.

## How to test this PR?
* Pull down the branch and visit site in browser to see new content & text changes
